src/fontra/views/editor/panel-reference-font.js

```diff
@@ -305,6 +305,20 @@ export default class ReferenceFontPanel extends Panel {
         this.requestReferenceFontsPreview();
       }
     );
+
+    editorController.sceneSettingsController.addKeyListener("location", () => {
+      const fontVariationSettings = [];
+      for (const axis of this.editorController.fontController.globalAxes) {
+        fontVariationSettings.push(
+          `'${axis.tag}' ${this.editorController.sceneSettings.location[axis.tag]}`
+        );
+      }
+      const buildFontVariablesCssString = fontVariationSettings.join(",");
```

Comment:
Suggest to use `cssString` as the variable name. The current name makes no sense.

Reply:
Ok, it's a renaming mistake.